app/views/layouts/application.html.erb

```diff
@@ -3,15 +3,19 @@
   <head>
     <title><%= full_title(yield(:title)) %></title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
     <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
   </head>
 
-  <body>
+  <body class="body">
+    <%= render(NavbarComponent.new()) %>
     <%= yield %>
-    <%= debug(params) if Rails.env.development? %>
+    <%= render(FooterComponent.new()) %>
+
+    <%# <%= debug(params) if Rails.env.development? %>
```

Comment:
gì đây

Reply:
Ý chị hỏi là dòng debug phải k ạ?
Dòng đó bạn Long dùng để in cái debug lên màn hình ạ.